packages/tailwindcss/src/compat/default-theme.ts

```diff
@@ -0,0 +1,23 @@
+import { Theme } from '../theme'
+import { createCompatConfig } from './config/create-compat-config'
+
+let theme = new Theme()
+
+export default {
+  ...createCompatConfig(theme).theme,
+  fontSize: {
+    base: ['1rem', { lineHeight: '1.5rem' }],
+  },
+  spacing: {
+    2: '0.5rem',
+    3: '0.75rem',
+    4: '1rem',
+    10: '2.5rem',
+  },
```

Comment:
Should these move into the `createCompatConfig` function? I understand this won't have _values_ then which maybe is the point but perhaps we can do something like this: https://github.com/tailwindlabs/tailwindcss/pull/14221/files#r1726670761

Reply:
Hmm in `createCompatConfig` this would be of type bare so we would need to "excute" the bare fn handler for the known spaces which is awkward again. 

maybe copy pasting is fine for now lol but we should make sure we catch all values not just `2`, `3`, `4`, and `10`

Reply:
I'm not sure if hardcoding the values is the way to go but the list needs to at least be expanded on. I thought about using createCompatConfig + resolveConfig here but I think that'll introduce some complexity. Given that this is really for backwards compatibility I think it's *probably* fine to hardcode them.